### ISSUE TITLE
Status tag in page explorer should have a font size in rem rather than pixels

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#9363]
### Issue Summary

In our Pages explorer menu, we use a status tag `c-status` component to display the pages’ publication state and locales. Unfortunately this component sets a font size in pixels – which doesn’t support font-size resizing in browsers.

Here is the component in question:

![page-explorer-status](https://user-images.githubusercontent.com/877585/195900319-e86ddd4a-a08d-4e7e-b275-47f9bed50373.png)

### Steps to Reproduce

1. Change your browser settings to have a larger default font size
2. Open the Wagtail admin
3. Open the Pages explorer

### Proposed solution

Here is where the font size is defined: https://github.com/wagtail/wagtail/blob/4ec77247282808735d4997a94ab8a237d3829d84/client/src/components/PublicationStatus/PublicationStatus.scss#L5

We need to change this to use a declaration in `rem` (pixel value divided by 16). So this would become:

```scss
.c-status {
  @apply w-bg-black-50 w-tracking-tight w-text-white-80;
  // stylelint-disable-next-line property-disallowed-list
  text-transform: uppercase;
  font-size: 0.625rem;
}
```

**Comment by thibaudcolas** [comment]
Assigning this one to @maryojo.

**Comment by maryojo** [comment]
Alright @thibaudcolas , I'd start working on this

**Comment by leumasxxx** [comment]
@thibaudcolas can i work on this

**Comment by Seremba** [comment]
Hi @leumasxxx it is assigned to @maryojo

**Comment by leumasxxx** [comment]
okay thanks @Seremba but id still like to get  first issue to work on

**Comment by maryojo** [comment]
I'm working on this already @leumasxxx

**Comment by leumasxxx** [comment]
@maryojo how can i get an issue to work on

**Comment by thibaudcolas** [comment]
@leumasxxx could we keep the discussion here to what this issue is about? If you’re looking for support with contributions please either use our Slack workspace, or GitHub Discussions, or contact me directly

**Comment by leumasxxx** [comment]
> @leumasxxx could we keep the discussion here to what this issue is about? If you’re looking for support with contributions please either use our Slack workspace, or GitHub Discussions, or contact me directly

okay

**Comment by Seremba** [comment]
Hi, @thibaudcolas I read somewhere that Mary had taken on another issue instead of this one! Can I go with it?

**Comment by Seremba** [comment]
Oh sorry! Thanks a lot @Scotchester

**Comment by lb-** [comment]
Fixed via #9382 4cf0cd62399288fe1941eaf73b880fdae3bef666
